contact.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Contact</title>
+	<meta charset="utf-8">
+	<meta content="width=device-width, initial-scale=1.0" name="viewport">
+	<link href="https://fonts.googleapis.com/css?family=Arsenal" rel="stylesheet">
+	<link href="index.css" rel="stylesheet/less" type="text/css">
```

Comment:
on line 8 the link tag's attributes are a little off, it should be 

```html
<link href="css/index.css" rel="stylesheet" type="text/css">
```

in order for the styling to take effect